### PR TITLE
⬆️  celery 5.2.7

### PR DESCRIPTION
**Pull request by ndittren**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -15,7 +15,11 @@ python-dateutil==2.8.2
 vine==1.3.0
 amqp==2.6.1
 kombu>=4.6.11,<5
```
Comment (nikolas) [inline requirements.txt:17]: According to this document kombu needs to be upgraded to version 5 here - https://docs.celeryq.dev/en/stable/history/whatsnew-5.0.html#kombu

I'm not really sure what kombu does.

**Comment by nikolas** [comment]
It looks like there are a few more changes required here, e.g. the two files that import from `celery.tasks.schedules` needs to replaced with `celery.schedules`. See: https://docs.celeryq.dev/en/stable/history/whatsnew-5.0.html#removed-deprecated-modules

**Comment by ndittren** [comment]
@nikolas ah missed the graphite file

**Comment by nikolas** [comment]
Also, while not explicitly required yet, it looks like a few of our `CELERY_*` configuration settings should be updated. See:
* https://docs.celeryq.dev/en/stable/history/whatsnew-5.0.html#step-2-update-your-configuration-with-the-new-setting-names
* https://docs.celeryq.dev/en/stable/userguide/configuration.html#conf-old-settings-map

File: requirements.txt [hunk 2]
```diff
@@ -14,8 +14,12 @@ python-dateutil==2.8.2
 
 vine==1.3.0
 amqp==2.6.1
-kombu>=4.6.11,<5
-celery>=4.4.7,<5.0.0
+kombu>=5.3.0b2,<6.0
```
Comment (nikolas) [inline requirements.txt:17]: We might as well just pin kombu to the exact version 5.3.0b2 for now, as 6.0 isn't out yet.

**Comment by ndittren** [comment]
> Also, while not explicitly required yet, it looks like a few of our `CELERY_*` configuration settings should be updated. See:
> 
> * https://docs.celeryq.dev/en/stable/history/whatsnew-5.0.html#step-2-update-your-configuration-with-the-new-setting-names
> * https://docs.celeryq.dev/en/stable/userguide/configuration.html#conf-old-settings-map

@nikolas I was a little confused with this. These docs: https://docs.celeryq.dev/en/stable/django/first-steps-with-django.html said to use uppercase for Django.

**Comment by nikolas** [comment]
Also good to be aware of the command-line interface changes: https://docs.celeryq.dev/en/stable/history/whatsnew-5.0.html#new-command-line-interface

See the `salt_state/apps/celery4*.service` systemd config files in salt_state for how we're actually starting celery. Fortunately it looks like no changes are needed here.

At this point I might as well roll this out and test things on production. I'll revert if I run into any issues I can't fix.

**Pull request merged**